I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 11, in <module>
    
AttributeError: 'module' object has no attribute 'ScaleFactor'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/usr/share/renpy/renpy/bootstrap.py", line 281, in bootstrap
    renpy.main.main()
  File "/usr/share/renpy/renpy/main.py", line 401, in main
    game.context().run(node)
  File "/home/nish/script.rpyc", line 11, in script
  File "/usr/share/renpy/renpy/ast.py", line 896, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/usr/share/renpy/renpy/python.py", line 1485, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 11, in <module>
    
AttributeError: 'module' object has no attribute 'ScaleFactor'

Linux-4.4.1-2-ARCH-x86_64-with-glibc2.2.5
Ren'Py 6.99.8.959
Polymorphism 0.0
